server/src/test/java/org/elasticsearch/index/mapper/TextFieldTypeTests.java

```diff
@@ -10,18 +10,7 @@
 import org.apache.lucene.index.Term;
 import org.apache.lucene.queries.intervals.Intervals;
 import org.apache.lucene.queries.intervals.IntervalsSource;
-import org.apache.lucene.search.AutomatonQuery;
-import org.apache.lucene.search.BooleanClause;
-import org.apache.lucene.search.BooleanQuery;
-import org.apache.lucene.search.ConstantScoreQuery;
-import org.apache.lucene.search.FuzzyQuery;
-import org.apache.lucene.search.PrefixQuery;
-import org.apache.lucene.search.Query;
-import org.apache.lucene.search.RegexpQuery;
-import org.apache.lucene.search.TermInSetQuery;
-import org.apache.lucene.search.TermQuery;
-import org.apache.lucene.search.TermRangeQuery;
-import org.apache.lucene.search.WildcardQuery;
+import org.apache.lucene.search.*;
```

Comment:
Could you run `gradlew spotlessApply` on your branch, this doesn't match our import style

Reply:
> Could you run `gradlew spotlessApply` on your branch, this doesn't match our import style

I'm sorry, this is my fault. It's possible that my IDE performed some operations that I didn't anticipate and I didn't check this part carefully. Now, I have executed `gradlew spotlessApply` to adjust the relevant code style.